### PR TITLE
Better column ordering

### PR DESCRIPTION
**Pull request by jcracknell**
Before: icon, user, sha, message, time
After: icon, message, user, time, sha

commit-list
file-list

### CODE REVIEW

**Comment by tboerger** [comment]
I would prefer the sha more on the left side

**Comment by jcracknell** [comment]
Interesting. I find in my workflow that the SHA is generally the least useful piece of information.

The real issue I am trying to fix here is that I keep clicking on the username instead of the SHA in the commit list, and I based the design on GitHub's ordering. I think making the SHA the leftmost item would probably work as well.

**Comment by tboerger** [comment]
Hum, I'm really unsure about that :(

**Comment by jcracknell** [comment]
Well in the second example, the only things that have changed from the status quo are (a) the committer name has been moved rightwards, and (b) the green tag has been removed from the SHA.

Right now the primary (leftmost) action for the commit list takes you to the user profile, which is almost never what the user wants to do.

**Comment by tboerger** [comment]
@lunny @makhov your opinions for that?

**Comment by lunny** [comment]
No idea. Both are OK!

**Comment by tboerger** [comment]
@jcracknell sorry for the trouble, maybe you are right and the first proposal seems to be the best. Can you please update the pull request again and squash the commits? I think afterwards we can finally merge it.

**Comment by jcracknell** [comment]
@tboerger Ok, I did a `git push --force origin HEAD^:column-order` and that seems to have worked nicely.

File: public/ng/less/gogs/repository.less [hunk 1]
```diff
@@ -294,6 +294,10 @@
             max-width: 100%;
         }
     }
+		td.sha {
```
Comment (tboerger) [inline public/ng/less/gogs/repository.less]: Wrong indention

File: public/ng/less/gogs/repository.less [hunk 2]
```diff
@@ -605,16 +612,16 @@
...
-    }
-    .message {
-        width: 60%;
-        span {
-            max-width: 500px;
-        }
+    .date {
+        width: 120px;
     }
+		.sha {
```
Comment (tboerger) [inline public/ng/less/gogs/repository.less]: Wrong indention

File: templates/repo/commits_table.tmpl [hunk 3]
```diff
@@ -24,16 +25,26 @@
...
+										</a>
+                    {{else}}
+                    <img class="avatar-20" src="{{AvatarLink .Author.Email}}" alt=""/>
+                    {{end}}
+								</td>
+                <td class="message"><span class="text-truncate"><a rel="nofollow" href="{{AppSubUrl}}/{{$username}}/{{$reponame}}/commit/{{.Id}}">{{.Summary}}</a></span></td>
                 <td class="author">
                     {{if .User}}
-                    <img class="avatar-20" src="{{.User.AvatarLink}}" alt=""/>&nbsp;&nbsp;&nbsp;<a href="{{AppSubUrl}}/{{.User.Name}}">{{.Author.Name}}</a>
+										<a href="{{AppSubUrl}}/{{.User.Name}}">{{.Author.Name}}</a>
```
Comment (tboerger) [inline templates/repo/commits_table.tmpl]: Wrong indention

File: templates/repo/commits_table.tmpl [hunk 4]
```diff
@@ -24,16 +25,26 @@
...
                     {{else}}
-                    <img class="avatar-20" src="{{AvatarLink .Author.Email}}" alt=""/>&nbsp;&nbsp;&nbsp;{{.Author.Name}}
+										{{.Author.Name}}
```
Comment (tboerger) [inline templates/repo/commits_table.tmpl]: Wrong indention

**Comment by tboerger** [comment]
Beside the wron indention it looks fine to me. You have to rebase against develop branch anyway, else we can not merge it.

**Comment by jcracknell** [comment]
Ok, well the indentation is all over the place, so I reformatted everything to four spaces, which seems to be the general intention.

**Comment by tboerger** [comment]
Let's make some progress! @fuxiaohei are you fine with that? Or any objections?

**Comment by jcracknell** [comment]
I'm all for consistency in indentation (being the person who added a `.editorconfig` in the first place), but right now there is none and it seems a bit goofy to "fix" the indentation in one spot when it's not even really clear what the "right" indentation actually is.

If you prefer a different indentation convention I can reformat accordingly, or I can go back and try to change the individual modified files to match the specific file's convention, or you can drop the pull request entirely as it was a pretty small change to begin with.

**Comment by lunny** [comment]
@jcracknell, could you please re-pull request since so many files changed? We need some clear commits.

**Comment by tboerger** [comment]
Replaced with #219

**Pull request closed without merging**